words2num/lang_EN_US.py

```diff
@@ -127,6 +128,16 @@ def transition(self, token):
         return edge_fn(self, value)
 
 
+def compute_placevalues(tokens):
+    """Compute the placevalues for each token in the list tokens"""
+    pvs = []
+    for tok in tokens:
+        try:
+            pvs.append(placevalue(VOCAB[tok][0]))
+        except KeyError:
+            pvs.append(0)
```

Comment:
I would lean towards mimicking the logic of the other parts of the code. I think you could even just not catch the error too
```
raise ValueError("Invalid number word: "
                         "{0} in {1}".format(e, text))
``

Reply:
I actually wrote this to catch the word "point" and fill it in with value 0. I'll change it to an if statement to catch it instead because I don't want to accidentally cover up any errors.